releases/geneva/compose-files/docker-compose-geneva-mongo-no-secty-arm64.yml

```diff
@@ -284,109 +277,103 @@ services:
 #################################################################
 
   device-virtual:
-    image: edgexfoundry/docker-device-virtual-go-arm64:1.2.0
+    image: edgexfoundry/docker-device-virtual-go-arm64:1.2.1
     ports:
     - "127.0.0.1:49990:49990"
     container_name: edgex-device-virtual
     hostname: edgex-device-virtual
     networks:
-      edgex-network:
-        aliases:
-        - edgex-device-virtual
+      - edgex-network
     environment:
       <<: *common-variables
       Service_Host: edgex-device-virtual
+      Service_Timeout: "20000"
     depends_on:
       - consul
 #      - logging  # uncomment if re-enabled remote logging
       - data
       - metadata
 
   device-rest:
-    image: edgexfoundry/docker-device-rest-go-arm64:1.1.0
+    image: edgexfoundry/docker-device-rest-go-arm64:1.1.1
     ports:
       - "127.0.0.1:49986:49986"
     container_name: edgex-device-rest
     hostname: edgex-device-rest
     networks:
-      edgex-network:
-        aliases:
-          - edgex-device-rest
+      - edgex-network
     environment:
       <<: *common-variables
       Service_Host: edgex-device-rest
+      Service_Timeout: "20000"
```

Comment:
Increase service timeout can't solve the issue, so I am wondering it is worth to add here

Reply:
@lenny-intel we might be able to remove this in each Device Service

Reply:
@cloudxxx8 , I have removed these